apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/Utilities.cs

```diff
@@ -46,10 +46,10 @@ public static ByteString Concat(this ByteString left, ByteString right)
         public static IEnumerable<T> ValidateCollection<T>(
```

Comment:
Side note: It might be worth making all these methods internal - the class itself is, but in code reviews you can't always see the class declaration.

Reply:
Done